core/src/main/java/io/debezium/connector/cassandra/CassandraConnectorConfig.java

```diff
@@ -347,11 +347,22 @@ public static EventOrderGuaranteeMode parse(String value) {
             .withType(Type.STRING)
             .withDescription("The absolute path of the YAML config file used by a Cassandra node.");
 
+    public static final Field CASSANDRA_CLUSTER_NAME = Field.create("cassandra.cluster.name")
+            .withType(Type.STRING)
+            .withDescription("Name of Cassandra cluster.")
+            .withDefault("Test Cluster");
+
     public static final Field COMMIT_LOG_RELOCATION_DIR = Field.create("commit.log.relocation.dir")
             .withType(Type.STRING)
             .withValidation(Field::isRequired)
             .withDescription("The local directory which commit logs get relocated to once processed.");
 
+    public static final Field CASSANDRA_CDC_LOG_LOCATION = Field.create("commit.log.location")
```

Comment:
Is this used somewhere else than in tests? If not then it should not be exposed as connector config option.